docs/change-log/index.md

```diff
@@ -13,16 +14,186 @@
 - [Limitations](#limitations)
 - [Deprecations](#deprecations)
 
+## v1.3.0
+
+### Client/server matrix and prereqs to installing
+
+The MLRun server is now based on Python 3.9. It's recommended to move the client to Python 3.9 as well. 
+
+MLRun v1.3.0 maintains support for mlrun base images that are based on python 3.7. To differentiate between the images, the images based on
+python 3.7 have the suffix: `-py37`. The correct version is automatically chosen for the built-in MLRun images according to the Python version of the MLRun client (for example, a 3.7 Jupyter gets the `-py37` images).
+
+If you are installing on a Python 3.9 client, run these commands before installing:
+?????
+
+If you are installing on a CE Python 3.9 client, run these commands before installing:
+?????
```

Comment:
can be removed

Reply:
replaced with: 
MLRun is pre-installed in CE Jupyter.